modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -4341,14 +4341,14 @@ public List<FragmentEntryLinkModel> newFragmentEntryLinkModels(
 		List<FragmentEntryLinkModel> originalFragmentEntryLinkModels =
 			new ArrayList<>();
 
-		LayoutModel nonhiddenLayout = null;
+		LayoutModel nonhiddenLayoutModel = null;
 
 		String imageRenderNamespace = StringUtil.randomId();
 		String paragraphRenderNamespace = StringUtil.randomId();
 
 		for (LayoutModel layoutModel : layoutModels) {
 			if (!layoutModel.isHidden()) {
-				nonhiddenLayout = layoutModel;
+				nonhiddenLayoutModel = layoutModel;
```

Comment:
@slnn Should the `H` in `nonhidden` be upper case?

Reply:
No, SF does not allow **nonHiddenLayout**